tests/cases/standalone/common/function/json/json.result

```diff
@@ -36,15 +36,15 @@ SELECT json_path_exists(parse_json('[1, 2]'), 'null');
 +-----------------------------------------------------------+
 | json_path_exists(parse_json(Utf8("[1, 2]")),Utf8("null")) |
 +-----------------------------------------------------------+
-| false                                                     |
+|                                                           |
 +-----------------------------------------------------------+
 
 SELECT json_path_exists(parse_json('null'), '$.a');
 
 +--------------------------------------------------------+
 | json_path_exists(parse_json(Utf8("null")),Utf8("$.a")) |
 +--------------------------------------------------------+
-| false                                                  |
+|                                                        |
 +--------------------------------------------------------+
```

Comment:
I'm not quite sure about this. "null" is a valid json string and "$.a" is a valid path. For me, it should return false since the path does not exist in the json here.

Reply:
I thought so too. @WenyXu and I had a discusstion in another [PR](https://github.com/GreptimeTeam/greptimedb/pull/4864#discussion_r1812018609) on this, maybe we need to delve more into this.

Reply:
> I thought so too. @WenyXu and I had a discusstion in another [PR](https://github.com/GreptimeTeam/greptimedb/pull/4864#discussion_r1812018609) on this, maybe we need to delve more into this.

I checked the [docs of the databend](https://docs.databend.com/sql/sql-functions/semi-structured-functions/json-path-exists).

```
SELECT JSON_PATH_EXISTS(NULL, '$.a');

----
NULL
```

The test should use the `NULL`  instead of `parse_json("null")`